tests/e2e/footer.spec.ts

```diff
@@ -50,31 +50,31 @@ const footerLinks = [
 const footerSocialMediaLinks = [
   {
     linkName: 'Twitter',
-    linkAddress: 'https://twitter.com/KodaDot',
+    linkAddress: 'KodaDot',
```

Comment:
Why replace the full URL with a partial one?

Reply:
> Why replace the full URL with a partial one?

because of external link redirections
for example, when i click the Instagram link, it redirects me to https://www.instagram.com/kodadot.xyz/ for other ppl it goes to https://www.instagram.com/accounts/login/?next=https%3A%2F%2Fwww.instagram.com%2Fkodadot.xyz%2F